Files/GenericFileBrowser.xaml.cs

```diff
@@ -70,6 +70,15 @@ public GenericFileBrowser()
             }
 
             App.OccupiedInstance.instanceViewModel.PropertyChanged += ViewModel_PropertyChanged;
+
+            // QuickLook Integration
+            ApplicationDataContainer localSettings = ApplicationData.Current.LocalSettings;
+            var isQuickLookIntegrationEnabled = localSettings.Values["quicklook_enabled"];
+
+            if (isQuickLookIntegrationEnabled != null && isQuickLookIntegrationEnabled.Equals(false))
```

Comment:
I would have the visibility be a bool property and have the actual item in the context menu bound to it instead of controlling it here.